CRM/Case/BAO/Case.php

```diff
@@ -1212,9 +1212,20 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
    *
    */
   public static function getRelatedContacts($caseID, $skipDetails = FALSE) {
+    $caseRoles = array();
+    if (!$skipDetails) {
+      $caseInfo = civicrm_api('Case', 'getsingle', array(
```

Comment:
When I was using `civicrm_api3` it caused a fatal in the syntaxConformanceTests. I'm not exactly sure why, but maybe due to the way they use transactions it was not able to find the case?